docker-compose.yml

```diff
@@ -12,7 +12,9 @@ services:
       context: .
     networks:
       -  faf-stack
-    
+    logging:
+      driver: "json-file"
```

Comment:
is this still needed?

Reply:
Will check and remove in next pr if not.

Reply:
already removed it, so you need to add it again... sry :)